### PR TITLE
Fix a couple of data races

### PR DESCRIPTION
**Pull request by tribal-tec**

### CODE REVIEW

File: plugins/RocketsPlugin/RocketsPlugin.cpp [hunk 1]
```diff
@@ -301,6 +307,33 @@ class RocketsPlugin::Impl
...
+        std::lock_guard<std::mutex> lock(_delayedNotifiesMutex);
+
+#ifdef BRAYNS_USE_LIBUV
+        if (uvw::Loop::getDefault()->alive())
+#else
+        if (false)
+#endif
+        {
+#ifdef BRAYNS_USE_LIBUV
+            // dispatch delayed notify from throttle thread to main thread
```
Comment (karjonas) [inline plugins/RocketsPlugin/RocketsPlugin.cpp]: I think it is more readable to make this one ifdef/else block.

File: tests/addModelFromBlob.cpp [hunk 2]
```diff
@@ -236,18 +227,15 @@ BOOST_AUTO_TEST_CASE(send_wrong_number_of_bytes)
...
     const std::string wrong("not_four_bytes");
     getWsClient().sendBinary(wrong.data(), wrong.size());
 
     try
     {
-        asyncWait.get();
+        while (!request.is_ready())
+            process();
+        request.get();
+        ;
```
Comment (karjonas) [inline tests/addModelFromBlob.cpp]: remove

File: plugins/RocketsPlugin/RocketsPlugin.cpp (hunk 1 above)
Comment (tribal-tec) [reply plugins/RocketsPlugin/RocketsPlugin.cpp]: yeah, agreed

File: tests/addModelFromBlob.cpp (hunk 2 above)
Comment (tribal-tec) [reply tests/addModelFromBlob.cpp]: done

**Pull request merged**